tests/test_cache.py

```
@@ -0,0 +1,131 @@
+    cache.set("key1", "value1")
+    cache.set("key2", {"nested": "data"})
+
+    stats = cache.get_stats()
+
+    assert stats['enabled'] is True
+    assert stats['file_count'] == 2
+    assert stats['size_mb'] >= 0  # Size can be 0.0 for very small files
+    assert stats['max_size_mb'] == 100
+
+
+def test_cache_size_enforcement(cache, tmp_path):
```

Ruff (0.15.1)

[warning] Unused function argument: `tmp_path`

(ARG001)